tests/ops/service/connectors/test_saas_queryconfig.py

```diff
@@ -50,9 +52,17 @@ def test_external_reference_traversal(
             in customer.parents.keys()
         )
 
+    @mock.patch(
+        "fides.api.ops.models.privacy_request.PrivacyRequest.get_cached_identity_data"
```

Comment:
We didn't need the identity to return the appropriate `read` request before but now it's needed so I mocked the call to the cache for the identity data